README.rst

```diff
@@ -87,155 +129,156 @@ must be present and contain a dict with Zyte API parameters:
+settings to change which headers are mapped and how they are mapped:
 
-        custom_settings = {
-            "ZYTE_API_DEFAULT_PARAMS": {
-                "geolocation": "US",  # You can set any Geolocation region you want.
-            }
-        }
+-   ``ZYTE_API_UNSUPPORTED_HEADERS`` determines headers that *cannot* be mapped
+    as ``customHttpRequestHeaders``, and its default value is:
 
-        def start_requests(self):
-            yield scrapy.Request(
-                url="http://quotes.toscrape.com/",
-                callback=self.parse,
-                meta={
-                    "zyte_api": {
-                        "browserHtml": True,
-                        "javascript": True,
-                        "echoData": {"some_value_I_could_track": 123},
-                    }
-                },
-            )
+    .. code-block:: python
 
-        def parse(self, response):
-            yield {"URL": response.url, "HTML": response.body}
+       ["Cookie", "User-Agent"]
 
-            print(response.raw_api_response)
-            # {
-            #     'url': 'https://quotes.toscrape.com/',
-            #     'statusCode': 200,
-            #     'browserHtml': '<html> ... </html>',
-            #     'echoData': {'some_value_I_could_track': 123},
-            # }
+-   ``ZYTE_API_BROWSER_HEADERS`` determines headers that *can* be mapped as
+    ``requestHeaders``. It is a ``dict``, where keys are header names and
+    values are the key that represents them in ``requestHeaders``. Its default
+    value is:
 
-            print(response.request.meta)
-            # {
-            #     'zyte_api': {
-            #         'browserHtml': True,
-            #         'geolocation': 'US',
-            #         'javascript': True,
-            #         'echoData': {'some_value_I_could_track': 123}
-            #     },
-            #     'download_timeout': 180.0,
-            #     'download_slot': 'quotes.toscrape.com'
-            # }
+    .. code-block:: python
 
-There is a shortcut, in case a request uses the same parameters as
-defined in the ``ZYTE_API_DEFAULT_PARAMS`` setting, without any further
-customization - the ``zyte_api`` meta key can be set to ``True`` or ``{}``:
+       {"Referer": "referer"}
 
-.. code-block:: python
+To maximize support for potential future changes in Zyte API, automated
+parameter mapping allows some parameter values and parameter combinations that
+Zyte API does not currently support, and may never support:
 
-    import scrapy
+-   ``Request.method`` becomes ``httpRequestMethod`` even for unsupported_
+    ``httpRequestMethod`` values, and even if ``httpResponseBody`` is unset.
 
+    .. _unsupported: https://docs.zyte.com/zyte-api/usage/extract.html#zyte-api-set-method
 
-    class SampleQuotesSpider(scrapy.Spider):
-        name = "sample_quotes"
+-   You can set ``customHttpRequestHeaders`` or ``requestHeaders`` to ``True``
```

Comment:
What do you think about adding a few examples? The reason I'm asking is that by this time it might not be clear where to set requestHeaders to True. Is it `settings`, `request.meta`, `request.meta['zyte_api']` or `request.meta['zyte_api_automap']`?

Reply:
I will be adding 2 code examples prior to this, to cover the default mapping behavior and the changes that e.g. `"screenshot": True` introduces.

But I am hesitant about including examples of this specific bullet point, or any sibling bullet point. These bullet points describe how automated parameter mapping responds to stuff that Zyte API does not currently support, and while I think it needs to be covered for completeness, I think it is OK if users do not fully understand this part, as they are not meant to be using it anyway.

When future Zyte API changes cause some of these bullet points to be relevant for actual Zyte API usage, then we can move those bullet points out of here into a different list, above, and include an example for them.